cmd/zed/lake/use/command.go

```diff
@@ -78,34 +78,39 @@ func (c *Command) Run(args []string) error {
 		return err
 	}
 	defer cleanup()
-	var branchName, baseName string
+	var poolName, branchName, baseName string
 	switch len(args) {
 	case 0:
 		if c.poolName == "" {
+			head, err := c.lakeFlags.HEAD()
+			if err == nil {
+				fmt.Printf("HEAD at %s\n", head)
+				return nil
+			}
 			return errors.New("a branch name or commit ID must be given")
```

Comment:
Nit: Indent the error path.
```suggestion
			if err != nil {
				return errors.New("a branch name or commit ID must be given")
			}
			fmt.Printf("HEAD at %s\n", head)
			return nil
```